Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM rocker/r-ver:4.0.2
+
+USER root
+
+RUN Rscript -e 'install.packages("remotes")'
+
+COPY DESCRIPTION /bound/DESCRIPTION
+RUN Rscript -e 'remotes::install_deps("/bound")'
+
+COPY . /bound
```

Comment:
TODO: Test if a single call to COPY really affects docker performance (re: caching installed packages)

Reply:
My experiments suggests it does make a difference. Thanks Jackson!